searches/interpolation_search.go

```diff
@@ -0,0 +1,47 @@
+package main
```

Comment:
It should belong to `package searches`.

Reply:
I noticed that every file on the searches folder is in the `main` package, should i leave mine there too or make it a separate package?

Reply:
It could imply to make a separate folder of the same name inside searches (as far as i know).
I can:
1. Change package of the go files inside the folder to searches
2. Leave my function in the main package the avoid import problems

Reply:
It's better to change it to `package searches` and the other search implementation should be changed to `package searches` too.